modules/ROOT/pages/provision-adapter-configure-service-mesh.adoc

```diff
@@ -11,6 +11,18 @@ The adapter behaves as the policy enforcement component, which fetches policies
 
 After you provision the adapter, you must redeploy all your existing applications to ensure that the Envoy sidecar is injected within each pod in the Kubernetes cluster.
 
+== Methods for Provisioning the Adapter
+
+Based on your requirements, you can provision the adapter using either of the following methods:
+
+* <<provision_the_adapter, Provision the adapter using Anypoint Service Mesh CLI>>
++
+If you want to create an adapter for a simple HTTP API, or do not manage you Kubernetes objects using custom resource definitions (CRD) files, use the CLI to provision the adapter.
```

Comment:
"If you want to create an adapter for a simple HTTP API," - What does this mean?

Reply:
removed "simple". Thanks!